CMake/ctkWrapPythonQt.py

```diff
@@ -1,9 +1,15 @@
+""" ctkWrapPythonQt
+
+Script to parse C++ headers and extract list of derived Qt classes
```

Comment:
Thanks for adding the docstring :+1: 

It turns out I was incorrect in my initial suggestion. Here is an improve documentation:

```
CLI allowing to generate PythonQt decorator for constructor and destructor of classes.

The CLI expects one class per header file with a class name matching the filename. 

It generates the files `<namespace>_<target>.h` and `<namespace>_<target>_init.cpp`
where:
* `<namespace>` is a string like `org_commontk`
* `<target>` is a string like `CTKWidgets`
```